platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/PlatformEvent.java

```diff
@@ -353,7 +353,7 @@ public void setConsensusTimestampsOnPayloads() {
         if (this.consensusData == NO_CONSENSUS) {
             throw new IllegalStateException("Consensus data must be set");
         }
-        final ConsensusTransactionImpl[] transactions = unsignedEvent.getTransactions();
+        final PayloadWrapper[] transactions = unsignedEvent.getTransactions();
```

Comment:
if we can get rid of the array and just use a list that would be great

Reply:
done